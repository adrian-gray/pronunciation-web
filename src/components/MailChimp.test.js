import React from "react";
import { render, cleanup } from "@testing-library/react";

import MailChimp from "./MailChimp";

afterEach(cleanup);

test("MailChimp", async () => {
  const props = {};

  const { container } = render(<MailChimp {...props} />);

  expect(container).toMatchSnapshot();
});
