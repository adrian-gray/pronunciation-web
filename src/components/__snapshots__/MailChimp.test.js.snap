// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MailChimp 1`] = `
<div>
  <div
    class="sub-canvas mailchimp-bg"
  >
    <div
      class="mailchimp-subscribe"
    >
      <div
        class="mailchip-root"
      >
        <form
          class=""
        >
          <div
            class="form-row"
          >
            <label
              class="form-label"
            >
              Your first name
            </label>
            <input
              autocomplete="name"
              class="form-control form-bottomspace form-control"
              id="signin_email"
              placeholder="Your first name"
              value=""
            />
          </div>
          <div
            class="form-row"
          >
            <label
              class="form-label"
            >
              Your Email
            </label>
            <input
              autocomplete="email"
              class="form-control form-bottomspace form-control"
              id="email"
              placeholder="Your Email"
              value=""
            />
          </div>
          <div
            class="form-row"
          >
            <br />
            <p
              class="form-divider"
            >
              We hate spam and will never share your details with anyone else.
            </p>
            <br />
            <button
              class="btn btn-warning"
              type="button"
            >
              Keep me updated!
            </button>
          </div>
        </form>
        <br />
      </div>
    </div>
  </div>
</div>
`;
